_specs/ecip-1000.md

```diff
@@ -73,10 +73,10 @@ A good reason to transfer ownership is because the original author no longer has
 The current ECIP editors are:
 
 * Wei Tang (@sorpaas)
-* Isaac Ardis (@meowsbits)
 * Cody Burns (@realcodywburns)
-* Afri Schoedon (@soc1c)
 * Yaz Khoury (@YazzyYaz)
+* @soc1c (Pseudonymous)
+* @meowsbits (Pseudonymous)
```

Comment:
On second thought, I'd prefer to assume a "fully qualified" pseudonym. Something regal and notable... something memorable... something inspiring....

```suggestion
* @meowsbits (Mr. Meows D. Bits)
```

Reply:
:tophat:

Reply:
@meowsbits How about `Mr. Meows D. Bits, Pseudonymous`? I just want to make sure people understand this is an anonymous tag, not a real name.

Reply:
Happy to break off into it's own scoped PR though. Not an immediate priority for me.

Reply:
But it will be.

Reply:
But it will be.